testng-core/src/test/java/testhelper/TestClassGenerator.java

```diff
@@ -0,0 +1,39 @@
+package testhelper;
```

Comment:
Please, use full package name (org.testng) for new classes

Reply:
I can refactor this package name to `org.testng.testhelper`. Is that what you are hinting at ?

Reply:
Yes, please :)

Reply:
@juherr fixed